src/sequences/SequenceEntity.js

```diff
@@ -14,6 +14,23 @@ export const sequencedActivityClasses = {
 };
 
 export class SequenceEntity extends Entity {
+	/**
+	 * @return An object with indices: completed, total, optionalViewed, optionalTotal, isCompleted
+	 */
+	completion() {
+		const subEntities = this._entity && this._entity.getSubEntitiesByRel && this._entity.getSubEntitiesByRel(Rels.completion);
+		const subEntity = subEntities && subEntities[0];
+
+		if (!subEntity) {
+			return {};
+		}
+
```

Comment:
nitpik: I don't think `subEntity` needs to be assigned when `subEntities` don't have substances. So, I would prefer to switch the order:
```
if (!subEnties) {
    return {};
}
const subEntity = subEntities[0];
```

Reply:
Okay